python/ray/data/preprocessor.py

```diff
@@ -245,19 +245,12 @@ def _transform(self, dataset: Dataset) -> Dataset:
                 "and 'arrow' Dataset formats are supported."
             )
 
-        transform_type = self._determine_transform_to_use(dataset_format)
```

Comment:
do we still need this function if it's caller is removed in `preprocessor.py` ? This is the only place we call into it within this file.

In addition, the other use of it is `BatchMapper` that we provide user the ability to decide a batch_format that overrides our default dispatch that should also be checked

Reply:
its still being called in `_transform_batch` right